httpd/hooks/reload

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Kill with a HUP signal on reload rather than restarting
```

Comment:
[nit] s/HUP/USR1/ in comment

Reply:
I opened this back when I had write access to this repo, but I don't any more, so I can't push to this branch. If somebody wants to push a commit fixing this that would be cool, or I can open another PR.

Reply:
@smith you're still listed as a maintainer and thus should still have access. If you would like to continue being a maintainer I can validate your permissions. Just say the word.

Reply:
@eeyun thanks for fixing that. @stevendanna this comment has been updated in 59009d2.